crypto/keys/der/der_test.go

```diff
@@ -68,9 +68,9 @@ func TestFromProto(t *testing.T) {
 			t.Parallel()
 
 			signer, err := FromProto(test.keyProto)
-			if gotErr := err != nil; gotErr != test.wantErr {
+			if (err != nil) != test.wantErr {
 				t.Fatalf("FromProto(%#v) = (_, %q), want (_, nil)", test.keyProto, err)
-			} else if gotErr {
+			} else if err != nil {
```

Comment:
Can get rid of the `else` now that there's no shared `gotErr` var.

Reply:
Reverted.